windows/deploy_vm/deploy_vm.yml

```diff
@@ -45,6 +45,7 @@
           include_tasks: ../../common/vm_take_screenshot.yml
           vars:
             vm_screenshot_local_dir: "{{ current_test_log_folder }}"
+            vm_screenshot_local_name: "after_gos_install.png"
```

Comment:
Maybe the screenshot file name can be same with Linux: `screenshot_at_deploy_success.png`

Reply:
Ok, changed, thanks.